test/end-to-end.js

```diff
@@ -961,6 +977,7 @@ describe('end-to-end', function() {
               if (err) return done(err);
               // localhost:3000 is the default value provided by loopback
               expectAppIsRunning('http://localhost:3000', done);
+              done();
```

Comment:
`expectAppIsRunning` is already accepting `done` callback. Please revert this change and fix `expectAppIsRunning` if needed.